i18n/zh/docusaurus-plugin-content-docs/current/plugin-center/proxy/divide-plugin.md

```diff
@@ -4,85 +4,142 @@ keywords: ["divide"]
 description: divide插件
 ---
 
-## 说明
+# 1. 概述
 
-`divide`插件是网关用于处理 `http协议`请求的核心处理插件。
+## 1.1 插件名称
 
-## 插件设置
+* `divide`插件
 
-* 引入相关依赖，开启插件，请参考：[Http快速开始](../../quick-start/quick-start-http) 。
+## 1.2 适用场景
 
-* `Http`应用客户端接入，请参考：[Http服务接入](../../user-guide/http-proxy) 。
+* 处理 `http协议` 请求
+* 支持流量治理，例如a/b 测试、灰度测试
+* 服务负载均衡
+* 设置接口的超时时间
 
-## 插件讲解
+## 1.3 插件功能
 
-`divide`插件是进行`http`正向代理的插件，所有`http`类型的请求，都是由该插件进行负载均衡的调用。
+* 支持根据 uri、header、query 等请求信息做流量的治理
+* 支持设置请求的负载均衡策略，同时支持服务预热，目前支持三种策略：ip hash（带虚拟节点的一致性哈希）、round-robbin（加权轮询）、random（加权随机）
+* 支持设置接口级别请求头最大值、请求体最大值、请求超时时间
+* 支持设置超时重试策略和重试次数，目前重试策略支持：current（重试之前失败的服务器）和failover（重试其它服务器）
 
-<img src="/img/shenyu/basicConfig/pluginHandle/selector_example.png" width="80%" height="80%" />
+## 1.4 插件代码
 
-客户端接入`Apache ShenYu`网关后，会自动注册选择器和规则信息，关于选择器和规则配置，请参考：[选择器和规则管理](../../user-guide/admin-usage/selector-and-rule)。
+* 核心模块 ```shenyu-plugin-divide```
+* 核心类 ```org.apache.shenyu.plugin.divide.DividePlugin```
 
+## 1.5 添加自哪个 shenyu 版本
 
-#### 选择器处理
+* 2.3.0
 
-<img src="/img/shenyu/basicConfig/pluginHandle/selector_add.png" width="80%" height="80%" />
+# 2. 如何使用插件
 
+## 2.1 插件使用流程图
 
-选择器处理: 对应[选择器和规则管理](../../user-guide/admin-usage/selector-and-rule)的`handle`字段，是网关匹配到流量以后，真实调用的`http`配置，可以配置多个，设置负载均衡权重，具体的负载均衡策略，在规则中指定。更多信息请参考插件管理中的 [插件处理管理](../plugin-handle-explanation) 。
+![](/img/shenyu/plugin/divide/procedure-cn.png)
 
-* 处理配置详解：
+## 2.2 导入 pom
 
-  * `host`：填写 `localhost`，该字段暂时没使用。
+- 在网关的 `pom.xml` 文件中添加插件 maven 配置。
 
-  * `ip:port`：`ip` 与端口，这里填写你真实服务的 `ip` + 端口。
+```xml
+  <dependency>
+      <groupId>org.apache.shenyu</groupId>
+      <artifactId>shenyu-spring-boot-starter-gateway</artifactId>
+     <version>${project.version}</version>
+  </dependency>
+```
 
-  * `protocol`：：`http` 协议，填写 `http://` 或者 `https://` ，不填写默认为:`http://`
+## 2.3 启用插件
 
-  * `startupTime`： 启用时间。
+- 在 `shenyu-admin` --> 基础配置 --> 插件管理 --> `divide` 设置为开启。
 
-  * `weight`：负载均衡权重。
+![](/img/shenyu/plugin/divide/enable-cn.png)
 
-  * `warmupTime`：预热时间。
+## 2.4 配置插件
 
-  * `status`：开启或关闭。
+### 2.4.1 配置客户端项目，在客户端项目的配置文件中配置
 
-  * `ip + port` 检测
+  * 服务和接口注册方式以及注册服务器地址
+  * 客户端配置，包含协议名称以及服务的路由地址
+  
+```yaml
+  shenyu:
+    register:
+      registerType: http #zookeeper #etcd #nacos #consul
+      serverLists: http://localhost:9095 #localhost:2181 #http://localhost:2379 #localhost:8848
+      props:
+        username: admin
+        password: 123456
+    client:
+      http: # http 协议
+        props:
+          contextPath: /http # 每个服务的路由地址
+```      
+
+### 2.4.2 在 shenyu-admin 配置文件中配置 upstream 有效性的检测参数
```

Comment:
Description This is the http register type used, users can also choose to use other register type（such as zookeeper..）